module/PowerShellEditorServices/Start-EditorServices.ps1

```diff
@@ -224,7 +224,7 @@ function Set-NamedPipeMode {
     )
     chmod $DEFAULT_USER_MODE $PipeFile
     if ($IsLinux) {
-        $mode = stat -c "%A" $PipeFile
+        $mode = stat -c "%a" $PipeFile
```

Comment:
Out of my depth here but I assume we know that %A works on macOS?  Also does the `else` below execute on Windows?  Or maybe we don't call this function at all on Windows?

Reply:
`stat -f "%a"` on macOS gives me something totally different

Reply:
`stat -f "%A"` gives me e.g. `700`